app/presenters/dc_time_presenter.rb

```diff
@@ -26,9 +26,11 @@ def convert
     time.in_time_zone(time_zone)
   end
 
-  def convert_and_format(format = FORMAT)
+  def convert_and_format(format = FORMAT, timezone_label: true)
```

Comment:
I figured since this is an admin page it would be a little annoying to see 6/1/2016 EST and such over and over again